recipe/custom_toolchain/cc_toolchain_config.bzl

```diff
@@ -307,6 +307,8 @@ def _impl(ctx):
 	    "${BUILD_PREFIX}/lib/gcc/${HOST}/${COMPILER_VERSION}",
 	    "${BUILD_PREFIX}/${HOST}/include/c++/${COMPILER_VERSION}",
             "${PREFIX}/include",
+            "/usr/local/cuda/include",
+            "/usr/local/cuda-${cuda_compiler_version}/include",
```

Comment:
Note that we pass in the variables somewhere explicitly. They are not automatically loaded from the environment.